docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -33,3 +33,31 @@ If you want to perform development tasks inside the {project} VM, it is recommen
 Depending on your operating system and shell environment, certain special characters can trigger variable interpolation and therefore cause passwords to fail.
 
 Workaround: When creating and entering passwords, wrap the string with single quotes in the following format: `'<password>'`
+
+
+[[authentication-required-to-push-image]]
+== Authentication required to push image to OpenShift integrated Docker registry
+
+If you are using Fedora, CentOS or RHEL host and you are running {project} with the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] 
```

Comment:
"If you are using Fedora, CentOS or RHEL host" 
-> If you are using a Fedora, CentOS or RHEL host
or 
-> If you are using Fedora, CentOS or RHEL 
or 
-> If you are using Fedora, CentOS or RHEL as host

Reply:
If you are using a Fedora, CentOS or RHEL host and you want to push to the OpenShift registry as described in xref:../openshift/openshift-docker-registry.adoc#[Accessing the OpenShift Docker Registry], you have to use the CentOS xref:../using/basic-usage.adoc#choosing-iso-image#[ISO]. 
Using the Boot2Docker or Minikube xref:../using/basic-usage.adoc#choosing-iso-image#[ISO] will result in the following authorization error:

....

The reason is an incompatibility between the Docker client shipped with your host and Docker daemon which is running inside the {project} VM.

Workaround: Use the xref:../using/basic-usage.adoc#choosing-iso-image#[CentOS ISO] when using {project} on Fedora,CentOS or RHEL.
For example:

Reply:
I am trying to split your mammoth sentence a bit.

Reply:
The rewrite @hferentschik shared is almost usable as-is. A couple of typos still exist which I will point out on their respective lines.